### PR TITLE
Create introduction/problems-solved.md

### PR DESCRIPTION
**Pull request by rrrutledge**
This file holds the text to accompany the [What Problems Does InnerSource Solve?](https://www.safaribooksonline.com/videos/introduction-to-innersource/9781492041504/9781492041504-video321607) video.
This accompanying text will appear as a part of the learning path (like [this example](https://www.safaribooksonline.com/learning-paths/learning-path-lean/9781491999738/9781491946527-/part01ch01.html)).

### CODE REVIEW

File: introduction/problems-solved.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+This file holds the text to accompany the [What Problems Does InnerSource Solve?](https://www.safaribooksonline.com/videos/introduction-to-innersource/9781492041504/9781492041504-video321607) video.
+This accompanying text will appear as a part of the learning path (like [this example](https://www.safaribooksonline.com/learning-paths/learning-path-lean/9781491999738/9781491946527-/part01ch01.html)).
+
+# What Problems Does InnerSource Solve?
+
+To illustrate the types of situations where inner source can help, imagine the following.
```
Comment (gruetter) [inline introduction/problems-solved.md]: I feel that InnerSource solves problems above and beyond the situation you described in this article. On a more abstract level, it fosters collaboration and learning where both was not possible beforehand due to strong siloing. The situation you describe here is "merely" one symptom of that problem. How about we lead with a more general statement along the lines of what I mentioned above and then follow with the concrete example you describe here.

File: introduction/problems-solved.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+For critical feature requests that are not prioritized for immediate work, the consuming team may commonly choose one of 3 options, each of which come with their own drawbacks.
+
+1. **Wait it out**. The consuming team may do nothing and limp along without the requested functionality.
+  This option requires the least amount of work on their side.
+  Depending on the benefit of the feature request, waiting might be just fine.
+  However, it may come with real amounts of pain, especially if the requested functionalty is never delivered.
+1. **Workaround**. A consuming team that doesn't want to wait may do extra work somewhere else to compensate for the absence of their requested feature.
+  This extra work may come as change in the consuming project.
+  Alternatively, they may create a new project that meets their needs and replaces their use of all or part of the producing team's system (code/project duplication).
+  This strategy allows the consuming team to get the requested feature via their own efforts only (no hard dependencies). It comes with several drawbacks, though.
```
Comment (gruetter) [inline introduction/problems-solved.md]: > (no hard dependencies)

Just wondering: what would be a _soft_ dependency?

File: introduction/problems-solved.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+      
+    1. Any work done by the consuming team remains unavailable to any other consumers with the same feature request.
+    1. The consuming team has inadvertently signed up for the long-term burden of maintaining the newly-written code, which is not in the domain of their core team competency.
+    1. The company as-a-whole acquires duplicate projects and code in the same problem space.
+      
+1. **Escalate**. The consuming team may not take "no" for an answer and, instead, advocate to someone in the producers' management hierarchy to influence (or force) the producing team to do the work.
+This option sounds attractive for the consuming team because they get the requested feature without doing the work to implement or maintain it.
+It is still a drag on the team, though, because it necessarily diverts some of their attention and work to the non-engineering task of escalation.
+Additionally, this option does not scale as there are so many times that a consumer can escalate feature requests before damaging their credibility.
```
Comment (gruetter) [inline introduction/problems-solved.md]: I'm not a native speaker, of course, but shouldn't we `s/there are so many times/there are _only_ so many times`?

File: introduction/problems-solved.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+Escalate is similarly disruptive (even more so) for the producing team, who are taken out of their normal workflow and prioritization methods to deal with the escalated feature request.
```
Comment (gruetter) [inline introduction/problems-solved.md]: s/Escalate/Escalation

File: introduction/problems-solved.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+This discussion sets the stage for inner source.
```
Comment (gruetter) [inline introduction/problems-solved.md]: Maybe here would be a good point where we could mention other problems that are solved very briefly, such as:

- lack of mentoring (often happens in PRs in InnerSource)
- lack of infrastructure and empowering to collaborate across silos
- lack of infrastructure to work asynchronously
- absence of technical career path (vs. meritocracy in InnerSource)

File: introduction/problems-solved.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
+This file holds the text to accompany the [What Problems Does InnerSource Solve?](https://www.safaribooksonline.com/videos/introduction-to-innersource/9781492041504/9781492041504-video321607) video.
```
[the remark this answers is not in the document]
Comment (rrrutledge) [reply introduction/problems-solved.md]: Thanks.

File: introduction/problems-solved.md (hunk 5 above)
Comment (rrrutledge) [reply introduction/problems-solved.md]: Thanks for these ideas.  These points sounds more like benefits to me as each one is phrased as the lack of a benefit rather than the presence of a problem.  Maybe we can call them out in [the benefits article](https://github.com/InnerSourceCommons/InnerSourceLearningPath/pull/5#issuecomment-454507604)?

Comment (rrrutledge) [reply introduction/problems-solved.md]: Added a short sentence summarizing the idea that there are other benefits to inner source besides what we've listed here.

File: introduction/problems-solved.md (hunk 4 above)
Comment (rrrutledge) [reply introduction/problems-solved.md]: Sounds good.

File: introduction/problems-solved.md (hunk 3 above)
Comment (rrrutledge) [reply introduction/problems-solved.md]: Apparently you are a better speaker than I am 😜

File: introduction/problems-solved.md (hunk 2 above)
Comment (rrrutledge) [reply introduction/problems-solved.md]: You're right, that term is not well-defined.  Removed it.

File: introduction/problems-solved.md (hunk 1 above)
Comment (rrrutledge) [reply introduction/problems-solved.md]: Great idea!  Added an extra paragraph at the beginning.

File: introduction/problems-solved.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
```
Comment (KathrynKLewis) [inline introduction/problems-solved.md]: Suggest rewording intro graf to be less wordy: 

Inner source encourages and rewards collaboration and code reuse with anyone, regardless of their position in a company's organizational structure. This approach differs what is seen in traditional organizations where ideas and work product tend to stay trapped within the boundaries of the internal corporate hierarchy and its silos.

File: introduction/problems-solved.md (hunk 5 above)
Comment (KathrynKLewis) [reply introduction/problems-solved.md]: I agree w Georg's comments. I think this intro should more clearly describe other problems that inner source solves so that we are capturing the reader's interest and curiosity to learn more.  One element that I think ought to be mentioned is as follows: 
Inner source also allows teams to work more efficiently by taking advantage of other software already built by teams across their organization. Team members do not have to repeatedly reinvent the wheel or solve problems that have already been solved because they can access the related or similar work that other teams have already done. Teams can then focus on higher value problems by leveraging their colleagues' work product.

Comment (rrrutledge) [reply introduction/problems-solved.md]: Updated the final paragraph - does that incorporate those critical ideas?

File: introduction/problems-solved.md (hunk 7 above)
Comment (rrrutledge) [reply introduction/problems-solved.md]: I love reducing words!

Comment (gruetter) [reply introduction/problems-solved.md]: love this, @KathrynKLewis !

File: introduction/problems-solved.md (hunk 5 above)
Comment (gruetter) [reply introduction/problems-solved.md]: I still think that _mentoring_ and _learning_ should be mentioned explicitly. The first sentence in the final paragraph focuses on _exposure_ to new tech and people but doesn't mention mentoring at all. That is, in my experience, what sets InnerSource apart from traditional projects, where you are also exposed to new people and tech, when you switch projects from time to time. Just my 2 c

Comment (rrrutledge) [reply introduction/problems-solved.md]: Done.

File: introduction/problems-solved.md [hunk 8]
```diff
@@ -38,5 +38,5 @@ Inner source applies to the same type of situation where a consuming team is una
 Inner source provides a way for the teams to gain the benefits of _wait it out_, _workaround_, and _escalate_ without the associated drawbacks.
 
 Inner source also provides a general improvement to engineering culture as engineers have the chance to work with a wider variety of new technologies and people.
-It facilitates sharing of ideas and solutions across organizational silos.
+Developers mentor and learn from one other as they share ideas and solutions across organizational silos.
```
Comment (lenucksi) [inline introduction/problems-solved.md]: Should this not be 'learn from one **an**other'?
Other than that, 👍.

**Comment by rrrutledge** [comment]
We wrote the last paragraph together after the video was shot.  Perhaps when we re-shoot the `problems solved` video we can re-shoot this one, too.
